### PR TITLE
initial commit of MLR-Validator

### PR DESCRIPTION
**Pull request by kmschoep-usgs**

### CODE REVIEW

File: insert_schema.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+def get_schema():
```
Comment (mbucknell) [inline insert_schema.py]: Can we call the module, 'schema'? Module and package names should in general be nouns

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+cerberus==1.1
```
Comment (mbucknell) [inline requirements.txt]: You should do a env/bin/pip freeze and put that in here. Pretty sure there are other dependencies brought in by Flask and flask-restplus

File: services.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    "transactionType": fields.String()
+})
+
+success_model = api.model('SuccessModel', {'success_message': fields.String()})
+
+error_model = api.model('ErrorModel', {
+    'error_message': fields.String()
+})
+
+parser = reqparse.RequestParser()
```
Comment (mbucknell) [inline services.py]: You aren't using this, so it can be removed.

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      description='MLR Validator Microservice',
+      author='Mary Bucknell, Andrew Yan, Dave Steinich, Zack Moore, Kathy Schoephoester',
+      author_email='mlr-devs@usgs.gov',
+      include_package_data=False,
+      long_description =read('README.md'),
+      install_requires=requirements,
+      test_loader='unittest:TestLoader',
+      platforms='any',
+      zip_safe=False,
+      py_modules=['app', 'config', 'services', 'validator_utils']
```
Comment (mbucknell) [inline setup.py]: You'll need to fix to have "validator" and "insert_schema" (or whatever name you change this to).

File: insert_schema.py (hunk 1 above)
Comment (kmschoep-usgs) [reply insert_schema.py]: I was going to call them "insert_schema", meaning the schema for an insert operation, and "update_schema" to handle the update validations.  Or should I just have 'schema.py' and have 'get_insert_schema()' and 'get_update_schema()'?

Comment (mbucknell) [reply insert_schema.py]: That makes sense to me.

**Pull request merged**